pkg/sloop/webfiles/sloop_ui.js

```diff
@@ -5,16 +5,34 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-const displayMaxX = document.documentElement.clientWidth;
-const displayMaxY = document.documentElement.clientHeight;
+const palette = {
+        baseDark: ["#2E3440", "#3B4252", "#434C5E", "#4C566A"],
```

Comment:
I'll throw in another lovely shade of white.

Reply:
I assumed these arrays were a collection of colors that were all used - either dark mode or light mode. After reading more code, it looks like only one color from each set is actually used. This seems misleading to me. I'd rather see this as just the single value that is being used. Perhaps the others could be kept as comments

Reply:
It's being kept for future guidance - in case more colors need to end up being used. Light isn't being used right now - but the various shades of dark should be used for whatever UI components are being created going forward.